src/Extension.php

```diff
@@ -41,6 +42,23 @@ public function load(ContainerBuilder $container, array $config): void
         $definition->addArgument($config['fileNamePrefix']);
         $definition->addArgument($config['outputDir']);
 
+        // Integration with other Behat extensions that provide screenshot services.
+        $imageUploaderService = null;
+        if (!empty($config['screenshotExtension'])) {
+          $tags = $container->findTaggedServiceIds('screenshot.service');
```

Comment:
I did not check the referenced package too deeply, but I imagine that this service tag is introduced by the other package?

Reply:
Yes, I tried to come up with a generic solution
For reference https://github.com/elvetemedve/behat-screenshot/pull/62/files#diff-367a0438232ae30182922c4e9618bd66be531f661c4e57d09b710749010b7fb9R23